app/controllers/api/v0/followers_controller.rb

```diff
@@ -0,0 +1,33 @@
+module Api
+  module V0
+    class FollowersController < ApiController
+      before_action :authenticate_with_api_key_or_current_user!, only: %i[organizations users]
+      before_action -> { limit_per_page(default: 80, max: 1000) }, only: %i[organizations users]
```

Comment:
I don't think you need the only clause, since there are only those two methods :)